src/DotNetty.Common/Utilities/ArrayExtensions.cs

```diff
@@ -72,5 +72,44 @@ public static void Fill<T>(this T[] array, int offset, int count, T value)
                 array[i] = value;
             }
         }
+
+        /// <summary>
+        ///     Merge the byte arrays into one byte array.
+        /// </summary>
+        public static byte[] CombineBytes(this byte[][] arrays)
+        {
+            int newlength = 0;
+            arrays.ForEach(delegate (byte[] array) { newlength += array.Length; });
```

Comment:
Hm. Sure. Add it as extension? And imma replace the ```int sum``` to ```long sum```.
Actually, not sure what to do with IByteBuffer then, since you need to count the ReadableBytes, or keep the ForEach anyway or just put another foreach above in ```IByteBuffer CopiedBuffer(params IByteBuffer[] buffers)``` to count the length